src/prefect/tasks/jira/jira_service_desk.py

```diff
@@ -0,0 +1,117 @@
+from prefect import Task
+from prefect.client import Secret
+from prefect.utilities.tasks import defaults_from_attrs
+from typing import cast
+from typing import Any
+
+try:
+    from jira import JIRA
+except ImportError:
+    pass
+
+
+class JiraServiceDeskTask(Task):
+    """
+    Task for creating a Jira Service Desk customer request. For this task to function properly,
+    you need a Jira account and API token.  The API token can be created at: https://id.atlassian.com/manage/api-tokens 
+    The Jira account username ('JIRAUSER'), API token ('JIRATOKEN') can be set as part of a 'JIRASECRETS' object in Prefect Secrets. 
+    An example 'JIRASECRETS' object looks like this:
+    ```
```

Comment:
🤔 This code block breaks the API documentation https://deploy-preview-2431--prefect-docs.netlify.app/api/latest/tasks/jira.html#jiratask

Reply:
Hmmm.... Thanks Josh. I'll check that.

Reply:
Should be ok now